Reference/Templating/Macros/Xslt/45-XML-Schema.md

```diff
@@ -66,8 +66,8 @@ XSLT file from the old to the new schema:
 
 However if you wish to revert back to the old XML Schema you need to
 update the ```UseLegacyXmlSchema``` setting in the
-```config/UmbracoSettings.config``` file, [make sure to read this wiki
+```config/UmbracoSettings.config``` file, make sure to read this wiki
```

Comment:
I think we need more reprashing as the old wiki can not be found.  So you can not read the wiki page.  Removing the link alone is not enough.  :-)

Reply:
@dampee Do you think its better to just remove the entire section on reverting back to the old schema? Given that this "new" schema was introduced in Umbraco 4.5, it has been nearly 8 years (we're now in v7.12!) and I highly doubt there are people who have yet to update. Not to mention I can't find any reverting instructions/docs to replace this link 🤔